bot/src/whitelist.ts

```diff
@@ -1,20 +1,41 @@
 import { getCollateralConfigByType } from 'auctions-core/src/constants/COLLATERALS';
 import { WHITELISTED_COLLATERALS } from './variables';
 
-const validateWhitelist = function (whitelist: string[]) {
-    whitelist.forEach(collateralType => {
-        getCollateralConfigByType(collateralType);
+const validateWhitelist = function (whitelist: string[]): Promise<void> {
+    return new Promise((resolve, reject) => {
```

Comment:
I don't see anything asynchronous in this function, why did you decide return promise here?

Reply:
I don't know what I was doing there to be honest 😅. I removed it now!